packages/account/src/Sections/Security/SelfExclusion/article-content.jsx

```diff
@@ -40,20 +40,7 @@ const ArticleContent = ({ is_eu, toggleModal }) => {
         />,
         <Localize
             key={3}
-            i18n_default_text='You can also exclude yourself entirely for a specified duration. If, at any time, you decide to trade again, you must then contact our <0>Customer Support</0> to remove this self-exclusion. There will be a 24-hour cooling-off period before you can resume trading.'
-            components={[
-                <a
-                    key={0}
-                    className='link link--orange'
-                    rel='noopener noreferrer'
-                    target='_blank'
-                    href={getStaticUrl('/contact-us', { is_deriv_crypto })}
-                />,
-            ]}
-        />,
-        <Localize
-            key={3}
-            i18n_default_text='UK clients won’t be able to remove their self-exclusion until the set period has expired. If you wish to continue trading once your self-exclusion period expires, you must contact <0>Customer Support</0> by phone to uplift this self-exclusion.'
+            i18n_default_text='You can also exclude yourself entirely for a specified duration. Once you have set your self-exclusion date, you won’t be able to remove it until the set period has expired. If you wish to continue trading once your self-exclusion period expires, you must contact <0>Customer Support</0> by phone on +447723580049 to uplift this self-exclusion.'
```

Comment:
usually we wrap phone number in a link <a href='tel:phone_num'>phone_num</a>

Reply:
Understood, will be incorporated.